src/main/java/com/baro/auth/infra/jwt/JwtTokenDecrypter.java

```diff
@@ -0,0 +1,44 @@
+package com.baro.auth.infra.jwt;
+
+import com.baro.auth.exception.jwt.JwtTokenException;
+import com.baro.auth.exception.jwt.JwtTokenExceptionType;
+import com.baro.common.auth.AuthMember;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.JwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import javax.crypto.SecretKey;
+
+@RequiredArgsConstructor
+@Component
+class JwtTokenDecrypter {
+
+    private final JwtProperty jwtProperty;
+
+    AuthMember decrypt(String authHeader) {
+        String token = validateTokenType(authHeader);
+        SecretKey secretKey = Keys.hmacShaKeyFor(jwtProperty.secretKey().getBytes());
+        try {
+            return new AuthMember(Jwts.parser().verifyWith(secretKey).build().parseSignedClaims(token).getPayload()
+                    .get("id", Long.class));
```

Comment:
🌗 가독성이 안좋은 것 같아요!
```suggestion
            Long id = Jwts.parser().verifyWith(secretKey)
                    .build()
                    .parseSignedClaims(token)
                    .getPayload()
                    .get("id", Long.class);
            return new AuthMember(id);
```

https://tecoble.techcourse.co.kr/post/2020-06-02-law-of-demeter/